processing/src/main/java/org/apache/druid/query/QueryContexts.java

```diff
@@ -54,6 +55,10 @@
   public static final String JOIN_FILTER_REWRITE_ENABLE_KEY = "enableJoinFilterRewrite";
   public static final String JOIN_FILTER_REWRITE_VALUE_COLUMN_FILTERS_ENABLE_KEY = "enableJoinFilterRewriteValueColumnFilters";
   public static final String JOIN_FILTER_REWRITE_MAX_SIZE_KEY = "joinFilterRewriteMaxSize";
+  // This flag control whether a sql join query with left scan should be attempted to be run as direct table access
+  // instead of being wrapped inside a query. With direct table access enabled, druid can push down the join operation to
+  // data servers.
+  public static final String SQL_JOIN_LEFT_SCAN_DIRECT = "enableSQLLeftTableScanDirect";
```

Comment:
this should maybe be `enableJoinLeftTableScanDirect` to be more consistent with other join related query context parameters. I also think it should probably be documented, but none of the other join query context parameters are documented (not sure why), so not sure its a blocker for this PR.